include/functions.h

```diff
@@ -456,7 +456,7 @@ void func_800304DC(PlayState* play, ActorContext* actorCtx, ActorEntry* actorEnt
 void Actor_UpdateAll(PlayState* play, ActorContext* actorCtx);
 s32 func_800314D4(PlayState* play, Actor* actor, Vec3f* arg2, f32 arg3);
 void func_800315AC(PlayState* play, ActorContext* actorCtx);
-void func_80031A28(PlayState* play, ActorContext* actorCtx);
+void Actor_KillActorsWithUnloadedObject(PlayState* play, ActorContext* actorCtx);
```

Comment:
feel like theres a more elegant way to name this but I dont have any immiediate idea. Mainly want to remove the redundant use of the word "actor"
`Actor_CleanupBasedOnObject` `Actor_KillAllWithMissingObject`, idk

Reply:
Note it's used exactly once

I don't have much better suggestion than what it is, yours or `Actor_KillActorsWhichObjectIsNotLoaded` idk

I think it should have `Kill` though since it `Actor_Kill`s and there are the various renowned ways to "remove" actors

Reply:
`Actor_KillAllWithMissingObject` is probably decent then. it removes the redundant "actor" I didnt like, and I think abstracting it to "missing" instead of unloaded makes it sound a bit less verbose, gets the point across that the idea is to clean up the actor list for those whos object is now not there after changing rooms

Reply:
alright sounds good, I'll change it to that